### PR TITLE
[CAPK] Fix E2E test - use the right image

### PR DESCRIPTION
**Pull request by nunnatsa**
Restore the previous solution to injct the image under test, as this solution was wrong.

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift-priv/cluster-api-provider-kubevirt/openshift-priv-cluster-api-provider-kubevirt-main.yaml [hunk 1]
```diff
@@ -172,8 +172,32 @@ tests:
...
+
+        # make sure the patch worked
+        oc get deployment -n hypershift operator -o yaml | grep "${CAPK_IMAGE}"
+
+        VALID_IMAGE="false"
+        # verify the image in the manager pod
+        for i in $(seq 20); do
+          sleep 60
+          # wait until the old pods is deleted; eventually we need only 2 pods to run
+          if [[ $(oc get pods -n hypershift -l "app=operator,hypershift.openshift.io/operator-component=operator,name=operator" --no-headers | wc -l) != 2 ]]; then
```
Comment (davidvossel) [inline ci-operator/config/openshift-priv/cluster-api-provider-kubevirt/openshift-priv-cluster-api-provider-kubevirt-main.yaml:185]: we aren't guaranteed that these two pods are the most up to date.

Comment (nunnatsa) [reply ci-operator/config/openshift-priv/cluster-api-provider-kubevirt/openshift-priv-cluster-api-provider-kubevirt-main.yaml]: Fixed

**Pull request merged**